md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/TableColumnLineageAPI.java

```diff
@@ -35,15 +40,35 @@
      * This method is used to see the relationship between processes linked to a particular table name and/or column name.
      * It also generates the dot string for visualisation of the links.
      *
-     * @param tableName, colName
-     * @return restWrapper It contains an instance of LineageInfo.
+     * @param lineageTabColInfo
+     * @return restWrapper It contains an instance of LineageTabColInfo.
      */
     @RequestMapping(value = {"", "/"}, method = RequestMethod.GET)
     public
     @ResponseBody
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)